src/Liquid/Tag/TagPaginate.php

```diff
@@ -106,7 +106,11 @@ public function __construct($markup, array &$tokens, FileSystem $fileSystem = nu
 	 */
 	public function render(Context $context)
 	{
-		$this->currentPage = (is_numeric($context->get('page'))) ? $context->get('page') : 1;
+		// Check for a setting in the Liquid config for the context key to use to refer to the current page
+		$pageContextKey = Liquid::get('PAGINATION_CONTEXT_KEY');
+		$pageContextKey = ($pageContextKey) ? $pageContextKey : 'page';
```

Comment:
There's `Liquid::$config` where defaults are. Better have them in one place, what do you think?